ui/StatusQ/src/StatusQ/Core/Utils/AmountsArithmetic.qml

```diff
@@ -99,6 +92,17 @@ QtObject {
         }
     }
 
+    /*!
+      \qmlmethod AmountsArithmetic::fromExponent(exponent)
+      \brief Construct a number 10^exponent in a safe manner.
+     */
+    function fromExponent(exponent) {
+        console.assert(Number.isInteger(exponent))
```

Comment:
Should we check if exponent is <= Big.MAX_POWER ?

Reply:
I woudn't go too far with those checks. Even much smaller values make no sense for our purposes.